### PR TITLE
Milestone : Project #7 Completed (Bootstrap) [Newsweek Homepage]

### PR DESCRIPTION
**Pull request by simonpeterdebbarma**
# Completed [Project #7- Bootstrap/Newsweek](https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap) in partial requirements of the [Microverse Curriculum](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/65938290/). 💯💯💯

---

### Contributors 👨🏼‍💻
- Simon Debbarma

--- 

##### General Requirements

- [x] Add descriptive README file to your repository.
- [x] Use meaningful names for your Pull Requests and add a short summary of its content.
- [x] Use descriptive commit messages.
- [x] Do not commit files that are not related to the project. Instead, add them to .gitignore file.
- [x] Do not commit inline comments. 
- [x] Use stickler for checks.

##### Other Requirements
- [x] Keep your code clean.
- [x] Remember to use semantic HTML.
- [x] Validate your HTML markup.
- [x] Use stylelint to lint CSS files.

##### Project Specific
- [x] Remember to use Bootstrap CSS classes wherever it is possible - this is the main point of this project.
- [x] As you can use a ready set of CSS classes, be more precise and make sure that your page looks exactly the same as the source page.
- [x] Remember to double-check that your page layout does not break on smaller screens

---

# Project Status :

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,20 +3,1858 @@
...
   <body>
+    <header class="block">
+      <div class="page-header">
+        <div class="container">
+          <div class="header-top flex-xs">
+            <div class="header-top-left">
+              <div class="header-date header-elements">Mon, Jan 06, 2020</div>
+            </div>
+           <div class="header-top-right">
+             <a href="" class="header-sign-in" >
```
Comment (pelzolga123) [inline index.html]: href is missing `#`

**Comment by pelzolga123** [comment]
## Hi Simon!  
### Your project looks :+1: amazing! You did great job making this project look almost like original :1st_place_medal: 

Here few things to correct before project can be approved:
 
* [ ] Check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows errors in your project.
* [ ] Footer part is a little messed:
image
Original:
image

* [Optional] You can use the `main` tag for the dominant content of the `body` of a document.
### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: index.html (hunk 1 above)
Comment (simonpeterdebbarma) [reply index.html]: Fixed

**Comment by izaiasneto4** [comment]
### Hi @simonpeterdebbarma. Nice job implementing the required changes. Project Approved :+1: 

TSE: Izaias Neto

**Pull request merged**